themes/dot/layouts/partials/head.html

```diff
@@ -34,5 +34,5 @@
 
     {{ "<!-- Bootstrap JS -->" | safeHTML }}
     <script src="{{ `plugins/bootstrap/bootstrap.min.js` | absURL }}"></script>
-
+    <script src="https://unpkg.com/lunr/lunr.js"></script>
```

Comment:
Is this needed? Looks like it's already included in the base.

Reply:
It's included where?

Reply:
https://github.com/JayNewstrom/recipes/pull/22/files#diff-254214e3367925e23471dec21f0e74327ee44551e2242f3c5e00902a87752a5aR25

Reply:
If it's a different script above, I'd prefer to host this file. All other assets are loaded from my domain.

Reply:
I'll grab a copy of it.